_opensearch/rest-api/_script-apis/get-stored-script.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Get Stored Script
+parent: Script APIs
+grand_parent: REST API reference
+nav_order: 3
+---
+
+## Get stored script
+
+Retrieves a stored script.
+
+### Path parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| script | String | Stored script or search template name. Required.|
+
+### Query parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| cluster_manager_timeout | Time | Amount of time to wait for a connection to the master node. Optional, defaults to `30s`. |
+
+#### Sample request
+
+The following retrieves the `my-script` stored script.
+
+````json
+GET _scripts/my-script
+````
+
+#### Sample response
+
+The `GET _scripts/my-script` request returns the following fields:
+
+````json
```

Comment:
Could we get an actual response from your my-first-script for consistency?

Reply:
I have made this change.